File: aspnetcore/Models/Segment.cs
using System;
using System.Collections.Generic;

namespace aspnetcore.Models
{
    public partial class Segment
    {
        public long Idsegment { get; set; }
        public long PostIdpost { get; set; }
        public string Type { get; set; } = null!;
        public string? Text { get; set; }
        public string? Source { get; set; }

        public virtual Post PostIdpostNavigation { get; set; } = null!;
    }
}

File: phoenix/blog/lib/schema/segment.ex
defmodule Schema.Segment do
  use Ecto.Schema
  @primary_key {:idsegment, :id, []}

  schema "segment" do
    field :post_idpost, :integer
    field :type, :string
    field :text, :string
    field :source, :string
  end
end
